app/js/components/homePage/manageSettings.jsx

```diff
@@ -9,11 +9,109 @@
 import React from 'react';
 
 export default class ManageSettings extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      AppBaseURLValue: '',
+      AppFolderPathValue: '',
+      AppStoreURLValue: ''
+    };
+
+    this.onChange = this.onChange.bind(this);
+    this.onCancel = this.onCancel.bind(this);
+  }
+
+  componentWillMount() {
+    this.setState({
+      AppFolderPathValue: 'appdata\\owa',
```

Comment:
Why the backward slashes?

Reply:
The double backslash is used to escape the backslash slash.